models/curvefi/celo/curvefi_celo_pools.sql

```diff
@@ -0,0 +1,22 @@
+{{
+    config(
+        tags = ['dunesql'],
+        alias = alias('pools'),
```

Comment:
can you add schema property here too?

i believe right now, without it assigned, it'll default to `curvefi.pools` since it's under the `curvefi` directory, but no entry exists for `curvefi/celo/` in the dbt_project file

we also want to keep schema in the models themsevles moving forward, for more transparency (we'll clean up project-wide when time permits)

Reply:
makes sense, will add `curvefi_celo` schema